docs/Testing-presubmit-Engine-PRs-with-the-Flutter-framework.md

```diff
@@ -2,23 +2,31 @@
 
 This documentation describes how to run flutter/flutter presubmit checks on flutter/engine PRs before submitting them.
 
-# Overview
+## Overview
 
-1. Wait for all presubmit checks on your flutter/engine PR to be green.
-2. Determine the commit hash for your flutter/engine PR.
-3. Create and upload a flutter/flutter PR, (OR run tests locally).
-4. Wait for flutter/flutter presubmits/tests to run ☕.
+1. Enable the web SDK presubmits.
+2. Wait for all presubmit checks on your flutter/engine PR to be green.
+3. Determine the commit hash for your flutter/engine PR.
+4. Create and upload a flutter/flutter PR, (OR run tests locally).
+5. Wait for flutter/flutter presubmits/tests to run ☕.
+6. Edit `.ci.yaml` to uncomment the `runIf` block you commented out above.
 
-Step (1) is the usual flutter/engine workflow.
+## 1. Edit .ci.yaml
 
-# 2. The commit hash
+Edit [`.ci.yaml`](https://github.com/flutter/engine/blob/main/.ci.yaml) to comment out the `runIf:` block in `linux_web_engine`. This will ensure Flutter Web artifacts build, otherwise most framework tests will fail during precache.
+
+## 2. Wait
+
+Contemplate the nature of the universe or why it is that this workflow was necessary for your situation. Could tests have been added to the engine? If not, get youself some coffee and a cookie. You'll need them.
```

Comment:
❤️

Reply:
Perhaps a jar to keep the tears you will shed during this 3h process. 😂